### ISSUE TITLE
Removing gaps for irregular time data on DateTimeAxis #83

### ISSUE THREAD

**Issue by aforoughi1** [oxyplot/oxyplot#1879]
I have reviewed #83, and am wondering if a solution already exists? 
Can you provide a sample?

**Comment by VisualMelon** [comment]
Not really; I did some work a while back here: https://github.com/oxyplot/oxyplot/issues/1640

It was a bit ambitious and I never got around to doing the design work for integrating the important bits into the library.
